packages/core/helper-plugin/src/index.js

```diff
@@ -1,49 +1,9 @@
 import { getType, getOtherInfos } from './content-manager/utils/getAttributeInfos';
 
-// Contexts
```

Comment:
I think we have to export these still, because they might be used for whatever reason by plugin developers.

Reply:
Okay, I'll make sure they're all exported :)